vendor/github.com/nmstate/kubernetes-nmstate/api/shared/condition_types.go

```diff
@@ -50,10 +50,12 @@ func (conditions *ConditionList) Set(conditionType ConditionType, status corev1.
 
 	// If there is different status, reason or message update it
 	if condition.Status != status || condition.Reason != reason || condition.Message != message {
+		if condition.Status != status {
```

Comment:
We should not apply patches to vendor/, no?

Reply:
We are vendoring the API as a subpackage, so we need to sync changes done in API

Reply:
I see, thanks